src/ontology/mondo-edit.obo

```diff
@@ -535045,7 +535046,7 @@ xref: OMIM:619063 {source="MONDO:equivalentTo"}
 xref: UMLS:C5436723 {source="MEDGEN:1729504", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
 is_a: MONDO:0033885 {source="OMIM:619063"} ! mitochondrial complex IV deficiency, nuclear-type
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/40045 {source="OMIM:619063"}
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/40045
```

Comment:
the omim source was removed because this is a provisional gene to phenotpye/disease relationship 
https://omim.org/entry/619063

Reply:
Same as above: the gene is provisional in OMIM, but it is used in the definition, therefore it should be kept.